src/java.base/share/classes/java/text/DecimalFormatSymbols.java

```diff
@@ -59,9 +59,12 @@
  * of these symbols, you can get the {@code DecimalFormatSymbols} object from
  * your {@code DecimalFormat} and modify it.
  *
- * <p>If the locale contains "rg" (region override)
- * <a href="../util/Locale.html#def_locale_extension">Unicode extension</a>,
- * the symbols are overridden for the designated region.
+ * <p>The "rg" (region override), "nu" (numbering system), and "cu" (currency)
+ * {@code Locale} <a href="../util/Locale.html#def_locale_extension">Unicode
+ * extensions</a> are supported which may override values within the symbols.
```

Comment:
This can be replaced with `{@link Locale##def_locale_extension Unicode extensions}`. There are multiple locations for this, and it is obviously out of this PR's scope, it can be cleaned up with a separate issue.

Reply:
Filed https://bugs.openjdk.org/browse/JDK-8365175 for this.